packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -429,8 +433,10 @@ export default {
       .countRelations()
       .build();
 
-    const entityPromises = ids.map((id: any) => documentManager.findOne(id, model, { populate }));
-    const entities = await Promise.all(entityPromises);
+    const entityPromises = documentIds.map((documentId: any) =>
+      documentManager.findLocales(documentId, model, { populate, locale })
+    );
+    const entities = (await Promise.all(entityPromises)).flat();
```

Comment:
Was thinking about creating an `async.flatMap`  utility for cases like this, wdyt? do you think its worth in cases like this?